File: C/Daily Flash/Week1/Day6_Jan18/code_C/prog2.c
/*
 * Program 2 : Write a Program to print Sum of First 10 Natural Numbers.
Output: The s um of First 10 Natural Numbers : 55
*
*/
#include <stdio.h>

void main() {

	
	int sum = 0;
	
	for(int i=1 ;i <= 10 ;i++)
		sum += i;

	printf("Sum of first 10 integers = %d\n",sum);
}

File: PYTHON/Daily Flash/Week2/Day1_Jan21/code/prog2.py
'''
Program 2 : Write a Program t o print cu be of first 20 numbers .
Output:
Cube of 1: 1
Cube of 2: 8
Cube of 3: 27
.
.
.
Cube of 20: 8000
'''

for x in range(20):
    print("Cube of "+ str(x+1) + ": " + str ( (x+1)* (x+1) * (x+1)))
